app/src/main/sentences/fr/util_yes_no.dslf

```diff
@@ -1,3 +1,3 @@
 util_yes_no: high
-[yes] oui|(pour sûr)|certainement|ok|okay|(Vas y)|affirmatif|procéder|continue|absolument|(fais le)|confirmation;
+[yes] oui|(pour sur)|certainement|(bien sur|entendu)|ok|okay|(vas y)|affirmatif|procéder|continue|absolument|(fais le)|(je? confirm<e|ation>);
 [no] non|stop|nope|arrête|ça suffit|assez|négatif|(ne le fais pas)|arrêt|retour;
```

Comment:
There are pre-existing missing parentheses around `ça suffit` (see #118). Could you fix it in this PR as well?

```suggestion
[no] non|stop|nope|arrête|(ça suffit)|assez|négatif|(ne le fais pas)|arrêt|retour;
```